src/pages/instant-observability.js

```diff
@@ -413,47 +351,97 @@ const QuickstartsPage = ({ data, location }) => {
             padding: var(--site-content-padding);
           `}
         >
-          <SuperTiles />
+          <div
+            css={css`
+              @media screen and (max-width: ${COLLAPSE_BREAKPOINT}) {
+                display: none;
+              }
+            `}
+          >
+            <SuperTiles />
+          </div>
           <div
             css={css`
               background-color: var(--secondary-background-color);
               border-radius: 4px;
-              padding: 1rem;
+              padding: 0.5rem;
               display: flex;
               justify-content: space-between;
               align-items: center;
 
               input {
                 font-size: 1.15em;
                 padding: 0.5rem;
-                padding-left: 3.75rem;
+                padding-left: 2.25rem;
+                background: var(--color-white);
+                border: var(--color-neutrals-600);
                 border-radius: 4px;
 
                 &::placeholder {
-                  color: var(--border-color);
+                  color: var(--color-neutrals-600);
+                  padding-left: 0.5rem;
                 }
               }
 
               .dark-mode & {
                 background-color: var(--tertiary-background-color);
-              }
-
-              @media screen and (max-width: 1180px) {
-                flex-direction: column;
-                align-items: normal;
+                input {
+                  background: var(--color-dark-500);
 
-                > * {
-                  margin: 0.5rem 0;
+                  &::placeholder {
+                    color: var(primary-text-color);
+                  }
                 }
               }
+              @media (max-width: ${COLLAPSE_BREAKPOINT}) {
+                background-color: var(--primary-background-color);
+              }
             `}
           >
             <SearchInput
               size={SearchInput.SIZE.LARGE}
               value={search || ''}
-              placeholder="Search for any quickstart (e.g. Node, AWS, LAMP, etc.)"
+              placeholder="What do you want to monitor? (e.g., AWS, LAMP, Kubernetes)"
               onClear={() => setSearch('')}
               onChange={(e) => setSearch(e.target.value)}
+              css={css`
+                --svg-color: var(--color-neutrals-700);
+
+                svg {
+                  width: 16px;
+                  height: 16px;
+                  color: var(--svg-color);
+                }
+
+                .dark-mode & {
+                  --svg-color: var(--primary-text-color);
```

Comment:
We most likely want to add some different background color in darkmode. The current color needs more contrast with the placeholder text:
<img width="1391" alt="Screen Shot 2022-01-03 at 2 12 52 PM" src="https://user-images.githubusercontent.com/70179303/147986212-03536b0d-40b6-487d-a419-0eeabe52bc62.png">

Reply:
update: 
<img width="750" alt="Screen Shot 2022-01-03 at 3 18 16 PM" src="https://user-images.githubusercontent.com/56952981/147990636-062d0b1f-3d1a-4a68-be1c-dedca173e1cf.png">

Does this work better?

Reply:
Looks good to me!